amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -4018,6 +4230,139 @@ definitions:
     properties:
       reason:
         type: string
+  RetentionNodeBodyCreate:
+    type: object
+    properties:
+      authority:
+        type: string
+      instructions:
+        type: string
+      isRecordLevel:
```

Comment:
We should have a description for this field, because it's not immediately obvious what the impact of sending true and false is. True will cause the the retention schedule to apply to records and false will cause the retention schedule to apply to record folders. This cannot be changed once items start being managed by the retention schedule.

Reply:
Updated the description as per the comment.